src/main.cpp

```diff
@@ -59,6 +59,10 @@ Adafruit_DRV2605 driver;
 // Select the desired effect, for now test effect "Buzz 100%"
 const uint8_t EFFECT = 14;
 
+// Select the alive publisher frequency:  20 (Hz) / _ALIVE_FREQUENCY_PUBLISHER_FLAG
+uint8_t _ALIVE_FREQUENCY_PUBLISHER_FLAG = 4;
+uint8_t _CURRENT_FLAG_VALUE = 0;
```

Comment:
The global values do not have to be preceded by an underscore. Also only `ALIVE_FREQUENCY_PUBLISHER_FLAG` should be upper case and made `const`. This way it is consistent with the rest of the code.
```suggestion
const uint8_t ALIVE_FREQUENCY_PUBLISHER_FLAG = 4;
uint8_t current_flag_value = 0;
```